README.md

```diff
@@ -1,11 +1,73 @@
+
+* Easily understand code sections
+* Tab to autocomplete code suggestions
+* Refactor functions where you are coding
+* Ask questions about your codebase
+* Quickly use documentation as context
+
+For more details, refer to [continue.dev](https://github.com/continuedev/continue)
+
+#### Models and Workflow of Vscode-Granite
+
+vscode-granite uses the Granite Code model, which is optimized for enterprise software development workflows and performs well across a range of coding tasks (e.g., code generation, fixing, and explanation), making it a versatile "all-around" code model.
+Granite Code comes in a wide range of sizes to fit your workstation's available resources. Generally, the bigger the model, the better the results.
+
+**Recommendation:** Model Size 8b for chat, 8b for tab code completion.
+For more details, refer to [Granite Code Models](https://github.com/ibm-granite/granite-code-models)
+
+##### Workflow of Vscode-Granite
+   ![workflow](media/workflow.svg)
+#### Installation Prerequisites:
+  * OS: Cross Platform 
+  * DISK SPACE :Minimum 30 GB 
+  * latest [Visual Studio Code](https://code.visualstudio.com/)
+#### Use Vscode-granite UI to install extention and models
+
+This project provides a user-friendly UI that simplifies the installation and management of extensions and Granite models. Developers can easily set up and configure their environment, and access the setup wizard anytime via the command palette by running "Granite: Setup Granite Code as code assistant".
+
+ step 1: **Install the Extenstion**
+
+Open VSCode and navigate to the Extension tab on the left sidebar. select on "vscode-granite" then click "install" to install the extension.
+ 
+ step 2:  **Install ollama**
+
+Once the extension is running, a new window will prompt you to install Ollama. 
+
+The [Continue.dev](https://continue.dev/) extension, if not already installed, will be automatically added as a dependency during this process. 
+
+you will be presented with the following installation options for installing ollama :
+
+1. **Install with Homebrew**: If Homebrew is detected on your machine (Mac/Linux).
+2. **Install with Script**: Available on Linux.
+3. **Install Manually**: Supported on all platforms( If you choose to install Ollama manually, you will be redirected to the official [Ollama download page](https://ollama.com/download) to complete the installation process).
+
+Once Ollama is installed, the page will refresh automatically.
+
+![installollama](media/installollama.gif)
+step 3: **Install granite models** 
+
+ select the Granite model you wish to install. Follow the on-screen instruction to complete the setup of your models.
+![installmodels](media/installmodels.gif)
```

Comment:
the gif is stale, we should see the option to keep existing settings in the combo